src/components/views/dialogs/SpotlightDialog.tsx

```diff
@@ -269,6 +291,9 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", onFinished }) =>
         return results;
     }, [possibleResults, trimmedQuery]);
 
+    const numResults = trimmedQuery ? people.length + rooms.length + spaces.length : 0;
```

Comment:
because I'm paranoid that JS will screw it up:
```suggestion
    const numResults = trimmedQuery ? (people.length + rooms.length + spaces.length) : 0;
```